compiler/src/iree/compiler/Codegen/Common/VectorizeMemrefCopy.cpp

```diff
@@ -104,12 +111,18 @@ struct VectorizeMemrefCopyPass final
     patterns.add<TileLinalgCopy>(&getContext());
     patterns.add<linalg::CopyVectorizationPattern>(&getContext());
     patterns.add<ConvertLinalgCopyToMemrefCopy>(&getContext());
+    // Try to remove generated single iteration loops and canonicalize generated
+    // subview operations.
+    populateRemoveSingleIterationLoopPattern(
+        patterns,
+        [&](scf::ForOp forOp) -> bool { return forOp->hasAttr(kIsTiled); });
```

Comment:
Would it make sense to add a lit test that checks for this directly?

Reply:
I added a test for this now.